azure/durable_functions/models/DurableOrchestrationClient.py

```diff
@@ -113,25 +113,43 @@ def create_check_status_response(self, request, instance_id):
         }
         return func.HttpResponse(**response_args)
 
-    def get_client_response_links(self, request, instance_id):
+    def create_http_management_payload(self, instance_id: str) -> Dict[str, str]:
         """Create a dictionary of orchestrator management urls.
 
         Parameters
         ----------
-        request : HttpRequest
+        instance_id : str
+            The ID of the orchestration instance to check.
+
+        Returns
+        -------
+        Dict[str, str]
+            a dictionary object of orchestrator instance management urls
+        """
+        return self.get_client_response_links(None, instance_id)
+
+    def get_client_response_links(
+            self,
+            request: Optional[Any], instance_id: str) -> Dict[str, str]:
```

Comment:
The type hint here specifies Optional[Any], but the docs specify Optional[HttpRequest].

Reply:
Good catch, I wish VSCode would warn me about the mismatch. Will fix